addons/iot_box_image/overwrite_after_init/etc/lightdm/lightdm.conf

```diff
@@ -1,8 +1,7 @@
-
 [LightDM]
 user-authority-in-system-dir=true
 
-[SeatDefaults]
+[Seat:*]
```

Comment:
no impact, just removes a warning in logs, as we were using the old name for the section.